components/CheckDocumentation/vacuum/TxidWraparound.tsx

```diff
@@ -0,0 +1,97 @@
+import React from "react";
+
+import {
+  CheckDocs,
+  CheckGuidanceProps,
+  CheckTriggerProps,
+} from "../../../util/checks";
+import { useSmartAnchor } from "../../SmartAnchor";
+
+const TxidWraparoundTrigger: React.FunctionComponent<CheckTriggerProps> = ({
+  config,
+}) => {
+  const warning = config.settings["warning_threshold_pct"] as number;
+  const critical = config.settings["critical_threshold_pct"] as number;
+  return (
+    <p>
+      Detects when the transaction ID utilization on the server is more than{" "}
+      <code>{warning}%</code> and creates an issue with severity "warning".
+      Escalates to "critical" once the utilization exceeds{" "}
+      <code>{critical}%</code>. Resolves once the utilization falls below the
+      threshold.
+    </p>
+  );
+};
+
+const TxidWraparoundGuidance: React.FunctionComponent<CheckGuidanceProps> = ({
+  urls: { backendsUrl },
+}) => {
+  const Link = useSmartAnchor();
+  return (
+    <div>
+      <h4>Impact</h4>
+      <p>
+        The transaction ID usage is high. Postgres usually runs anti-wraparound
+        autovacuums to keep the usage low by freezing old rows. However, if such
+        VACUUMs are either not running or are running but unable to freeze rows
+        for some reasons, the transaction usage keeps growing.
+      </p>
+      <p>
+        The anti-wraparound VACUUM needs to happen at some point and this is not
```

Comment:
I think this confuses the issue: it's technically true that VACUUM needs to perform freezing (i.e., the "anti-wraparound" part), but this does not have to be the "vacuum (to prevent wraparound)" that doesn't cede locks. Ideally, all the freezing you need happens in the course of regular vacuums.

(Confusing the issue even further is that the current Postgres vacuum scheduling mechanism is poor, so in practice you'll often get into situations where you won't get freezing until an anti-wraparound vacuum happens, but that's a slightly different issue, and probably not something we need to get into here.)